src/game.cpp

```diff
@@ -4421,17 +4421,17 @@ void Game::updateGui(const RunStats &stats, f32 dtime, const CameraOrientation &
 		std::ostringstream os(std::ios_base::binary);
 		os << std::fixed
 			<< PROJECT_NAME_C " " << g_version_hash
-			<< ", FPS " << fps
+			<< ", FPS: " << fps
 			<< std::setprecision(0)
-			<< ", Drawtime " << drawtime_avg << "ms"
+			<< ", drawtime: " << drawtime_avg << "ms"
 			<< std::setprecision(1)
-			<< ", Dtime jitter "
+			<< ", drawtime jitter: "
```

Comment:
dtime jitter
But i can fix on merge.

Reply:
oops forgot that one :P, I could fix it real quick.